app/furniture/marketplace/order.rb

```diff
@@ -22,6 +22,11 @@ class Order < Record
       paid: "paid"
     }
 
+    monetize :payment_processor_fee_cents
+    def vendors_share
+      product_total - payment_processor_fee
+    end
+
```

Comment:
Should `product_total` be `monetize`d as well? It seems odd to be substracting two values, one of which is monetized and the other one isn't.

Reply:
product_total is monetized, in that it sums the monetized product_price * quantity